frontend/packages/ceph-storage-plugin/src/models.ts

```diff
@@ -23,5 +23,16 @@ export const OCSServiceModel: K8sKind = {
   namespaced: true,
   kind: 'StorageCluster',
   id: 'ocscluster',
+};
+
+export const CSVModel: K8sKind = {
```

Comment:
Not required, use the generic CSV model.

Reply:
ack

Reply:
Yet to be removed.